src/stores/process.ts

```diff
@@ -168,14 +168,32 @@ export function createCommandFactory<T, P extends object = DefaultPayload>(): Co
 export type Commands<T = any, P extends object = DefaultPayload> = (Command<T, P>[] | Command<T, P>)[];
 
 const processMap = new Map();
-const valueSymbol = Symbol('value');
+export const valueSymbol = Symbol('value');
```

Comment:
is there any way we can test this without exporting the symbol? seems like we're exporting an implementation detail?

Reply:
I'm not sure what we could do without leaking something out here. I agree it's not great, but Proxies are not supposed to expose that they are proxies, and it will look upon inspection just like the object it's proxying in this case.

Reply:
Would it be preferable if I exposed an `isProxy` check or something that kept the symbol internal? Or created a proxy util module that created the proxy and provided that `isProxy` check that both still used? I know that's still exposing some kind of implementation detail but at least they wouldn't have access to the symbol.

Reply:
yeah that might be nicer (something like `isStateProxy()`?, at least people wouldn't be able to tag things with the symbol then. maybe i'm just being overly cautious in the first place.

Reply:
Doesn't seem overly cautious to me. Exposing that symbol does open up the user to doing some bad things if they wanted to.

I've made that change now.